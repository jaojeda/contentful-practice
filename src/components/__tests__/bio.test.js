import { useStaticQuery } from "gatsby";
import React from "react";
import renderer from "react-test-renderer";

import Bio from "../bio";

beforeEach(() => {
    useStaticQuery.mockReturnValueOnce({
        site: {
        siteMetadata: {
          author: {
            name: "Jose Ojeda",
            summary: "who lives and works in Portland building useful things."
          },
          social: {
            twitter: "JosAOjed"
          }
        }
      }})
})

describe("Bio", () => {
    it("renders the bio section", () => {
        const bio = renderer
            .create(<Bio />)
            .toJSON();
        expect(bio).toMatchSnapshot();
    });
});