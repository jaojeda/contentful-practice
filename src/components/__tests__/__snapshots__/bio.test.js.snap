// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bio renders the bio section 1`] = `
<div
  className="bio"
>
  <p>
    Written by 
    <strong>
      Jose Ojeda
    </strong>
     
    who lives and works in Portland building useful things.
     
    <a
      href="https://twitter.com/JosAOjed"
    >
      You should follow them on Twitter
    </a>
  </p>
</div>
`;
